go.mod

```diff
@@ -44,7 +44,6 @@ require (
 	goji.io v2.0.2+incompatible
 	golang.org/x/oauth2 v0.0.0-20190604053449-0f29369cfe45
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
-	golang.org/x/xerrors v0.0.0-20190717185122-a985d3407aa7
```

Comment:
Quick Question: What are go.mod and go.sum and why do you need to change these when the real changes for this PR seem to just be logic changes in update_formulas.go?

Reply:
go.mod defines the dependencies for the project, go.sum is a lock file that fully enumerates all the dependencies required to build the module

if import statements change or pull in new versions in go code, it is expected to have go.mod and go.sum change

Reply:
as for why they changed in this PR despite no changes to import statements, my guess is that @GrahamGoudeau ran `go mod tidy` before committing, which should trim the dependencies imported for a module down to the required components (cleaning up any extra dependencies listed in the module files)

Reply:
Gotcha, thanks for the explanation.

Reply:
In other projects, we enforce that `go mod tidy` is run prior to commit. We don't do that here just because this repo doesn't have a clear owner. In particular, it will update the go.sum file (a cryptographic hash of downloaded module content, for security) with versions of packages published for all available build platforms. That's why a lot of the changes here are additive. And in other projects we enforce that any diff to generated files (like go.sum) should fail the CI run. So I ran it out of habit here, thus this diff.